lib/galaxy/managers/group_roles.py

```diff
@@ -2,10 +2,11 @@
 from typing import (
     Any,
     Dict,
-    List,
     Optional,
 )
 
+from lib.galaxy.schema.schema import GroupRoleListModel, GroupRoleModel
```

Comment:
```suggestion
from galaxy.schema.schema import GroupRoleListModel, GroupRoleModel
```
`galaxy` is the root here.